api/files/serializers.py

```diff
@@ -203,13 +204,26 @@ class BaseFileSerializer(JSONAPISerializer):
         'upload': WaterbutlerLink(),
         'delete': WaterbutlerLink(),
         'download': 'get_download_link',
+        'render': 'get_render_link',
+        'html': 'absolute_url',
```

Comment:
New - added html link for files - for node files this will resolve to file detail page on the OSF. for preprint files, we're already not showing file detail page for preprint files, this resolves to the preprint.